internal/services/gateway/gateway.go

```diff
@@ -320,6 +327,11 @@ func (g *Gateway) Run(ctx context.Context) error {
 	apirouter.Handle("/orgs/{orgref}/members", authForcedHandler(orgMembersHandler)).Methods("GET")
 	apirouter.Handle("/orgs/{orgref}/members/{userref}", authForcedHandler(addOrgMemberHandler)).Methods("PUT")
 	apirouter.Handle("/orgs/{orgref}/members/{userref}", authForcedHandler(removeOrgMemberHandler)).Methods("DELETE")
+	apirouter.Handle("/orgs/{orgref}/invitations", authForcedHandler(orgInvitationsHandler)).Methods("GET")
+	apirouter.Handle("/orgs/{orgref}/invitations", authForcedHandler(createOrgInvitationHandler)).Methods("POST")
+	apirouter.Handle("/orgs/{orgref}/invitations/{userref}", authForcedHandler(orgInvitationHandler)).Methods("GET")
+	apirouter.Handle("/orgs/{orgref}/invitations/{userref}", authForcedHandler(deleteOrgInvitationHandler)).Methods("DELETE")
+	apirouter.Handle("/orgs/{orgref}/invitations/{userref}/actions", authForcedHandler(orgInvitationActionHandler)).Methods("PUT")
```

Comment:
Since the invitation could be only processed by the authenticated user it should be something like `/user/org_invitations/{orgref}/actions`